gatsby/content/blog/2020/02/2020-02-24-synapse-1.11.0-release.mdx

```diff
@@ -0,0 +1,98 @@
+---
+date: '2020-02-24'
+title: Synapse 1.11.0 released
+categories:
+  - General
+author: Matthew Hodgson
+---
+
+Hi all,
+
+Synapse 1.11 landed on Friday (sorry for running late on blogging the release notes!)
+
+The main change is to introduce an experimental API [MSC2432](https://github.com/matrix-org/matrix-doc/pull/2432) for managing aliases for rooms on your local server.  In Synapse 1.10 we removed support for `m.room.aliases` events, which were a way to try to track which aliases a room had on the room itself (but were vulnerable to abuse).  In this release we've re-added the ability to query which aliases a given server has for the room, giving visibility for managing aliases, without having them spray all over the room state itself.  Riot/Web 1.5.10 supports the new API, giving a way to manage aliases on your local server while we finish off the remaining work to improve alias safety & maintenance.
+
+We've also changed the default power levels for rooms so that room upgrades and ACLs require you to be an Admin (PL100), and invites in public rooms now require you to be a moderator.
```

Comment:
```suggestion
We've also changed the default power levels for rooms so that room upgrades and ACLs require you to be an Admin (PL100), and invites in public rooms now require you to be a moderator (PL50).
```

Reply:
oops, we raced on that and i caught it - thanks tho